tests/validation/converters/test_pandapower_converter_input.py

```diff
@@ -116,3 +116,16 @@ def test_extra_info__serializable(extra_info):
 
     # Assert
     json.dumps(actual, cls=JsonEncoder)  # expect no exception
+
+
+@pytest.mark.filterwarnings("error:.*invalid value encountered in divide.*:RuntimeWarning")
+def test_pgm_input_lines__cnf_zero():
+    pp_network = pp_net_3ph_minimal_trafo()
+    pp_converter = PandaPowerConverter()
+    pp_network.line.c_nf_per_km = 0
+    data, _ = pp_converter.load_input_data(pp_network)
+    assert data[ComponentType.line]["tan1"] == 0
+    pp_network.line.c_nf_per_km = 0.001
+    pp_network.line.c0_nf_per_km = 0
+    data, _ = pp_converter.load_input_data(pp_network)
+    assert data[ComponentType.line]["tan0"] == 0
```

Comment:
shouldn't it do the opposite? as in: check that it does **not** raise this warning anymore?

I will have a quick look tomorrow morning

Reply:
done